tasks/pre-process-changelog.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3.6
+import os
+import re
+import subprocess
+import sys
+from pathlib import Path
+
+
+def main():
+    manipulate_the_news()
```

Comment:
worth having `main()` as just a trivial wrapper here? (personal opinion, feel free to ignore!) -- a simpler way to write this is `main = manipulate_the_news` ;)

Reply:
I can see how this can be taken in two directions: my personal opinion would be to not have this trivial wrapper and just rename the `manipulate_the_news` function -> `main`.  But of course feel free to ignore this comment :)

Reply:
I would also say, just alias it